torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -412,6 +424,7 @@ ExecutionPlan ProfilingGraphExecutorImpl::getPlanFor(
   ProfilingRecord::removeProfileCounter(copy->block());
   runProfilingOptimizations(copy);
   // cache
+  GRAPH_DUMP("Optimized Graph: ", copy);
```

Comment:
you added it back!!!!!!!!!!!!!!!! thank you!!!! ))))))))) I was missing this one sorely, since this is what I was grepping for